rbi/stdlib/rubygems.rbi

```diff
@@ -2710,12 +2728,24 @@ class Gem::Version < Object
   # ver2 = Version.create(ver1)       # -> (ver1)
   # ver3 = Version.create(nil)        # -> nil
   # ```
+  sig { params(input: T.any(NilClass, String, Gem::Version)).returns(T.nilable(Gem::Version)) }
   def self.create(input); end
 
   # Constructs a
   # [`Version`](https://docs.ruby-lang.org/en/2.7.0/Gem/Version.html) from the
   # `version` string. A version string is a series of digits or ASCII letters
   # separated by dots.
+  sig do
+    params(
+      version: T.any(
+        String,
+        Integer,
+        Float, # Technically, any `to_s`able object would work.
+        Gem::Version, # Will return the same version object
+        NilClass # Depracated, and will issue a warning
+      )
+    ).void
+  end
```

Comment:
This looks like a correct sig for `initialize`, but not for `self.new`, which should return a value (I think this is `T.self_type`?).

Reply:
Correct, that explains the typecheck failure in CI, too. Fixed!